src/components/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const Spinner: React.FC = () => (
```

Comment:
Use this https://github.com/sef-global/scholarx-frontend/blob/development/src/assets/svg/Loading.tsx